src/pykeen/models/inductive/inductive_nodepiece.py

```diff
@@ -163,14 +164,18 @@ def __init__(
             self.num_valid_entities = validation_factory.num_entities
             self.num_test_entities = test_factory.num_entities
 
-    def _entity_representation_from_mode(self, *, mode: Optional[InductiveMode]):
+    # docstr-coverage: inherited
+    def _get_entity_representations_from_inductive_mode(
+        self, *, mode: Optional[InductiveMode]
+    ) -> Sequence[Representation]:  # noqa: D102
         assert mode is not None, "Inductive mode must be explicitly set for inductive models"
```

Comment:
I'd rather see this explicitly raise an exception and have a modified docstring explaining this

Reply:
1e44ada